### PR TITLE
[User] Sign in page

### PR DESCRIPTION
**Pull request by truongha-1240**
## Related Tickets
- [#47486](https://edu-redmine.sun-asterisk.vn/issues/47486)

## WHAT (optional)
- Build login page 
## HOW
-i create .html file
## Evidence (Screenshot or Video)
image

image

### CODE REVIEW

**Comment by truongha-1240** [comment]
ready

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+<div class="user_login_class">
+  <div class="form_login">
+    <h1><%= t ".login" %></h1>
+    <%= form_for :session, url: login_path do |f| %>
+      <%= f.label :email %>
+      <%= f.email_field :email, :required => true %>
+      <%= f.label :password, t(".password_label") %>
+      <%= link_to t(".forgot_password"), "#" %>
+      <%= f.password_field :password, :required => true %>
```
Comment (ghost) [inline app/views/sessions/new.html.erb]:
```suggestion
      <%= f.password_field :password, required: :true %>
```

**Review by ghost — COMMENTED** [review]
cmt

File: app/views/sessions/new.html.erb (hunk 1 above)
Comment (truongha-1240) [reply app/views/sessions/new.html.erb]: updated

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**